engine/src/main/java/org/terasology/engine/modes/loadProcesses/CreateWorldEntity.java

```diff
@@ -92,13 +92,14 @@ public boolean step() {
             // Also set all the components to the world entity.
             WorldConfigurator worldConfigurator = worldGenerator.getConfigurator();
             Map<String, Component> params = worldConfigurator.getProperties();
+
             for (Map.Entry<String, Component> entry : params.entrySet()) {
                 Class<? extends Component> clazz = entry.getValue().getClass();
-                Component comp = config.getModuleConfig(generatorUri, entry.getKey(), clazz);
+                Component comp = gameManifest.getModuleConfig(generatorUri, entry.getKey(), clazz);
```

Comment:
The thing that bothered me about this (and drove me to try #4584 to see if I was understanding things correctly) is that it seems like this removes the only use of `Config` in here.

and I don't see anything in this PR that takes values out of Config and puts them in GameManifest.

is there someplace else that is moving values from Config to WorldConfigurator before they are copied from WorldConfigurator to GameManifest?